app/views/booths/index.html.haml

```diff
@@ -0,0 +1,50 @@
+.container
+  .row
+    .col-md-12.page-header
+      %h1
+        Your booth requests for
+        = @conference.title
+  .row
+    .col-md-12
+      .margin-booth-table
+        %table.table.table-striped.table-hover
+          %thead
+            %th
+              %b ID
+            %th
+              %b Logo
+            %th
+              %b Title
+            %th
+              %b State
+            %th
+              %b Actions
+          - @booths.each do |booth|
+            %tr
+              %td
+                = booth.id
+              %td
+                - if booth.logo_link
+                  = image_tag(booth.picture.thumb.url, width: '20%')
+              %td
+                = link_to booth.title, conference_booth_path(@conference.short_title, booth)
+              %td
+                = booth.state
```

Comment:
It looks like you are allowing users to see if their booth is marked as to_accept, or to_reject. However these states were supposed to be for organizers only, and users will be notified when their booth would be marked as accepted, or rejected, isn't that so?